src/main/java/org/apache/sysds/hops/UnaryOp.java

```diff
@@ -165,10 +167,25 @@ else if(_op == OpOp1.MEDIAN) {
 				}
 				else //default unary 
 				{
+					boolean inplace = false;
+
+					if (OptimizerUtils.ENABLE_UNARY_UPDATE_IN_PLACE) {
```

Comment:
Great check, please make it a method that can be called that return a boolean, and put that method in src/main/java/org/apache/sysds/hops/Hop.java.
call it something like:
canBeUpdateInPlace()